PlasmaResearch/plasma_batch_auction.tex

```diff
@@ -304,10 +305,19 @@ \subsubsection*{Price Determination}
 \end{itemize}
 \end{lemma}
 */
-Along with this price vector, the plasma operator would also generate for a batch a list of tuples representing order-volume pairs, 
-\[[(\sigma_1, \nu_1), (\sigma_2,  \nu_2), \dots, (\sigma_N, \nu_N)]\]
-where $(\sigma_i, \nu_i)$ denotes the $i^{th}$ order along with the volume $\nu_i$ of fulfilment and $N$ is the number of orders in the batch. 
-This list will be referred to as the \emph{volume-list}\label{volumeBlock} and the lists Merkle hash will be the \emph{volume-hash}.
+Along with this price vector, the plasma operator would also share the trading volumes in a compact manner. This will be done using a mapping $m$ from the order index to the trading volume\emph{volume-list}\label{volumeBlock}. But we would only store the trading volumes for fractional fulfilled orders.
```

Comment:
The \emph{volume-list} doesn't look right in the pdf here. Not exactly sure what was intended with this...

Also. This single line is actually three sentences (should be turned into three lines).